backend/src/crd/controller/scheduledworkflow/util/setup.go

```diff
@@ -0,0 +1,62 @@
+package util
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/cenkalti/backoff"
+	"github.com/kubeflow/pipelines/backend/src/common/util"
+	"github.com/pkg/errors"
+
+	api "github.com/kubeflow/pipelines/backend/api/go_client"
+)
+
+const (
+	addressTemp = "%s:%s"
+)
+
+func WaitForAPIAvailable(initializeTimeout time.Duration, basePath string, apiAddress string) error {
```

Comment:
nit: this duplicates https://github.com/kubeflow/pipelines/blob/cc83e1089b573256e781ed2e4ac90f604129e769/backend/src/common/util/service.go#L31.

Consider using that function directly?